src/app/components/content/IsaacCoordinateQuestion.tsx

```diff
@@ -16,6 +17,16 @@ interface CoordinateInputProps {
     readonly?: boolean;
     remove?: () => void;
 }
+
+export const coordinateInputValidator = (input: string) => {
+    const errors = [];
+    const operatorExpression = new RegExp(".*[0-9][+/÷-]\\.?[0-9]+");
```

Comment:
```suggestion
    const operatorExpression = new RegExp("[0-9]\s*[+\/÷\-x×]\s*[0-9]");
```
I think this regex can be simplified to this as:
* the leading consumer is not required
* multiplication may be a desired thing to warn about as well
* I wasn't sure why `\` was important to capture as well as the optional wild card
* We only need at least one number after the symbol

This additionally allows spaces around the binary operator symbol.

You can go further and use `/[0-9]\s*[+\/÷\-x×]\s*[0-9]/` instead of `new RegExp(...)`